### PR TITLE
Shifts from last Sunday and Monday

### PR DESCRIPTION
**Pull request by fsoubelet**
Tested locally, all good.

### CODE REVIEW

File: docs/resources/logbook/2024_lhc.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ The tables below can be sorted by clicking next to the column headers.
...
 | 2024-03-18 22:40 | 2024-03-19 07:00 |    1WN     | Commissioning |      Nonlinear Corrections at 30cm - 2      | [Shift Plan](2024-03-18, 1081, 3988476){.logbook-link} / [Summary](2024-03-19, 1081, 3988607){.logbook-link} |
 | 2024-03-22 07:00 | 2024-03-22 16:00 |     1W     | Commissioning |      Global Corrections in the Squeeze      | [Shift Plan](2024-03-22, 1081, 3990992){.logbook-link} / [Summary](2024-03-22, 1081, 3991469){.logbook-link} |
+| 2024-03-24 10:40 | 2024-03-24 19:15 |     1H     | Commissioning |          Measurements in the Ramp           | [Shift Plan](2024-03-24, 1081, 3992589){.logbook-link} / [Summary](2024-03-24, 1081, 3992822){.logbook-link} |
+| 2024-03-25 07:00 | 2024-03-12 01:30 |   0.3WN    | Commissioning |       Squeeze validation and NL optics      | [Shift Plan](2024-03-25, 1081, 3993108){.logbook-link} / [Summary](2024-03-25, 1081, 3993341){.logbook-link} |
+| 2024-03-25 07:00 | 2024-03-12 01:30 |   0.3WN    | Commissioning |       Squeeze validation and NL optics      | [Shift Plan](2024-03-25, 1081, 3993108){.logbook-link} / [Summary](2024-03-25, 1081, 3993341){.logbook-link} |
```
Comment (JoschD) [inline docs/resources/logbook/2024_lhc.md:34]: twice?

**Review by JoschD — CHANGES_REQUESTED** [review]

File: docs/resources/logbook/2024_lhc.md (hunk 1 above)
Comment (JoschD) [reply docs/resources/logbook/2024_lhc.md:34]: also the end-date looks wrong

Comment (fsoubelet) [reply docs/resources/logbook/2024_lhc.md]: keyboard shortcuts are hard

**Review by JoschD — CHANGES_REQUESTED** [review]
end-date/time still wrong

**Review by JoschD — APPROVED** [review]
I love it when it all works out on the first commit!

**Pull request merged**